Maquetado/index.html

```diff
@@ -9,13 +9,74 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
+<body class="body-container">
+  <div class="navbar">
+    <img src="./assets/wolox_logo.svg">
+    <ul>
+      <li class="nav-option">Developer Tools</li>  
```

Comment:
since this is just to show where you are you can use a `span` although in most cases what goes inside the `nav` are links that take you to the different parts of the app.